CHANGELOG.md

```diff
@@ -9,18 +9,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [`x.y.z`] - Unreleased
 
+### Breaking changes:
+- The condition for sending Spatial position updates has been changed, the two variables `PositionUpdateFrequency` and `PositionDistanceThreshold` have now been replaced in the GDK settings by 4 new variables. To update your project:
```

Comment:
nit - I'd say they've been removed but not specify 4 unless you are going to say what they are.
Saying - they are gone and here are the steps to restore functionality - is good.